homework-2/script/maim.js

```diff
@@ -0,0 +1,318 @@
+        let paginationBtn = document.createElement('li');
+            paginationBtn.classList.add('pagination-list');
+            navigationBtn.push( paginationBtn );
+            paginationBtn.textContent = i;
+            paginationPage.append( paginationBtn );
+        }
+
+        showPage(navigationBtn[0], data);
+        
+        navigationBtn.forEach((btn) => {
+            btn.addEventListener('click', function() {
+                showPage( this, data );
+            });
+        });
+    }
+
+    let activeList;
+    function showPage(item, data) {
+
+        dataCompany.splice(0, dataCompany.length);
+        dataName.splice(0, dataName.length);
+        dataAddress.splice(0, dataAddress.length);
+        dataIdUser.splice(0, dataIdUser.length);
+        
+        if ( activeList ) {
+            activeList.classList.remove('active-pagination');
+        } 
+            activeList = item;
+            item.classList.add('active-pagination');
+            
+        
+        let pageNum = +item.innerHTML;
+
+        let start = (pageNum - 1) * notesOnPage;
+        let end = start + notesOnPage;
+
+        let notes = data.slice(start, end);
+        notes.forEach(({ company, name, address, id }) => {
+            dataCompany.push( company );
+            dataName.push( name );
+            dataAddress.push( address );
+            dataIdUser.push( id );
+        });
+        renderUserInfo( notes );
+    }
+    
+
+    function renderUserInfo(value) {
+        tableUser.innerHTML = '';
+
+        value.forEach(( {company, name, address, city, country} ) => {
+            tableUser.innerHTML += `
+                <tr class="table-user-info">
+                    <td class="table-user-list">
+                    <label class="checkbox">
+                        <input class="tabel-checkbox" name="user-checkbox" type="checkbox">
+                        <span class="box-checkbox "></span>
+                    </label>
+                    </td>
+                    <td class="table-user-list">${company}</td>
+                    <td class="table-user-list">${name}</td>
+                    <td class="table-user-list">${address}</td>
+                    <td class="table-user-list">${city}</td>
+                    <td class="table-user-list">${country}</td>
+                    <td class="table-user-list"><i class="fas fa-pen"></i></td>
+                    <td class="table-user-list"><button type="button" class="delete-user"><i class="fas fa-times-circle"></i></button></td>
+                </tr>
+            `;
+            
+        }); 
+        
+        let changeDataBtn = document.querySelectorAll('.fa-pen');
+            deleteBtn = document.querySelectorAll('.delete-user');
+            secondaryCheckbox = document.querySelectorAll('.box-checkbox');
+            checkboxBtn = document.querySelectorAll('.tabel-checkbox');
+    
+        secondaryCheckbox.forEach(( item, index ) => {
```

Comment:
when all items are selected, the remove btn should appear in the table head
![image](https://user-images.githubusercontent.com/84564899/126711685-157703d8-873b-4e45-91cb-735f5bef5f98.png)

Reply:
Done!